draft-robert-privacypass-batched-tokens.md

```diff
@@ -161,7 +167,7 @@ Parameters:
   Scalar skS
   Element pkS
 
-def BlindEvaluateBatch(blindedElements):
+def BlindEvaluate(blindedElements):
```

Comment:
I changed the name here to make it clear that the function differs from the one in the VOPRF spec. Reading the spec again, I see that it states the following:

> In the description above, inputs to GenerateProof are one-item lists. Using larger lists allows servers to batch the evaluation of multiple elements while producing a single batched DLEQ proof for them.

I'm now wondering if we need the function at all (also Finalize). On the pro side, it might make sense to be more explicit here. In that case, we should probably mention that this is the same as in the VOPRF spec but for multiple inputs and only spelled out here for convenience.

Let me know what you think.

Reply:
Right, I read this as a convenience. We've waffled on the batch presentation in the VOPRF draft for a while now. Do you find this presentation awkward? Would it be better if the VOPRF APIs just naturally worked on batches of inputs?

Reply:
At the end of the day, all alternatives work. But since you ask, I think I would have a slight preference VOPRF APIs that support batching.

I think we can still move forward with the PR for now if we include a sentence explaining that this is an alternative representation of BlindEvaluate from VOPRF, spelled out for convenience. If VOPRF changes, we can make a new PR.